packages/component-library/package.json

```diff
@@ -21,7 +21,7 @@
     "!tsconfig.dist.json"
   ],
   "sideEffects": false,
-  "version": "17.6.2",
+  "version": "0.0.0",
```

Comment:
Why is this 0.0.0 not 1.0.0? Will the build process automagically take care of the bumping here? If so, should this PR instead be `BREAKING CHANGE:`?

Reply:
The last commit to the branch includes a `BREAKING CHANGE` annotation, so this will get released at `1.0.0`. It's a minor quirk of the release pipeline that there isn't another way to release packages at `1.0.0` — manually setting the major version will always lead to `1.0.1` or `1.1.0` for the initial release. :joy: